modules/virt-enabling-persistentreservation-feature-gate.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enabling-persistentreservation-feature-gate_{context}"]
+= Enabling the PersistentReservation feature gate
+
+You can enable the SCSI PersistentReservation feature gate and allow a LUN-backed block mode virtual machine (VM) disk to be shared among multiple virtual machines.
+
+The PersistentReservation feature gate is disabled by default.
+
+.Prerequisites
+
+* Cluster administrator privileges are required.
+* The volume access mode `ReadWriteMany` (RWX) is required if the VMs that are sharing disks are running on different nodes. If the VMs that are sharing disks are running on the same node, the `ReadWriteOnce` (RWO) volume access mode is sufficient.
+* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
+
+.Procedure
+
+. Enable the `persistentReservation` feature gate by running the following command:
```

Comment:
Earlier you have "PersistentReservation" feature gate with no backticks and a capital P - should these all be consistent?

Reply:
Done!